examples/pyspark_ml_autologging/one_vs_rest.py

```diff
@@ -1,10 +1,14 @@
+import os
+
 from pyspark.ml.classification import LogisticRegression, OneVsRest
 from pyspark.ml.feature import VectorAssembler
 from pyspark.sql import SparkSession
 from sklearn.datasets import load_iris
 
 import mlflow
 
+os.environ["PYSPARK_PIN_THREAD"] = "false"
```

Comment:
nit: shall we add a comment about the context?

Reply:
This was a test. Removed this line.